### PR TITLE
完善所有c#相关的文档和代码

### PR DESCRIPTION
**Pull request by haptear**
完善所有c#相关的文档和代码

### CODE REVIEW

File: codes/csharp/chapter_computational_complexity/leetcode_two_sum.cs [hunk 1]
```diff
@@ -0,0 +1,68 @@
+﻿using NUnit.Framework;
+using System;
```
Comment (krahets) [inline codes/csharp/chapter_computational_complexity/leetcode_two_sum.cs:2]: Please remove all the unused libs for every files.

File: codes/csharp/chapter_computational_complexity/leetcode_two_sum.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
+﻿using NUnit.Framework;
```
Comment (krahets) [inline codes/csharp/chapter_computational_complexity/leetcode_two_sum.cs:1]: Please add a file header for all the files.

**Review by krahets — COMMENTED** [review]
So efficient. 👍 👍  Thanks for the great work!

I've just updated the `chapter array and linkedlist`, please check it.
The function `list.Remove(3)` should be replaced by `list.RemoveAt(3)` (which has been updated in my commit.) I notice this mistake because the output of the c# code does not match the Java code. Please compare the output of the two languages carefully.

**Comment by haptear** [comment]
@krahets 
please review again

**Comment by krahets** [comment]
@haptear My commit https://github.com/krahets/hello-algo/pull/159/commits/c429e5f0bb583eac68a939e549ffeed8c6d2f1f2 is lost in the latest code. Please recover it.

**Review by krahets — COMMENTED** [review]
@haptear The commit https://github.com/krahets/hello-algo/pull/159/commits/c429e5f0bb583eac68a939e549ffeed8c6d2f1f2 has been lost in the latest code. Please recover it.

By the way, have you tested the code? The code could not print the tree correctly.

image

**Review by krahets — APPROVED** [review]
I've tested and fine-tuned the code.
Thanks for the great work! Now merge it.

**Comment by krahets** [comment]
By the way, could you add `deque.cs` which is the only untranslated code? Thank you!

**Comment by krahets** [comment]
image

I noticed your GitHub ID isn't listed in the contributors, because your GitHub ID doesn't match your local Git ID.

**Pull request merged**